rest-api/src/main/scala/za/co/absa/enceladus/rest_api/controllers/v3/RunControllerV3.scala

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.rest_api.controllers.v3
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.{HttpStatus, ResponseEntity}
+import org.springframework.security.core.annotation.AuthenticationPrincipal
+import org.springframework.security.core.userdetails.UserDetails
+import org.springframework.web.bind.annotation._
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder
+import za.co.absa.atum.model.{Checkpoint, ControlMeasureMetadata, RunStatus}
+import za.co.absa.enceladus.model.{Run, Validation}
+import za.co.absa.enceladus.rest_api.controllers.BaseController
+import za.co.absa.enceladus.rest_api.controllers.v3.RunControllerV3.LatestKey
+import za.co.absa.enceladus.rest_api.exceptions.{NotFoundException, ValidationException}
+import za.co.absa.enceladus.rest_api.models.RunSummary
+import za.co.absa.enceladus.rest_api.services.v3.RunServiceV3
+
+import java.net.URI
+import java.util.Optional
+import java.util.concurrent.CompletableFuture
+import javax.servlet.http.HttpServletRequest
+import scala.concurrent.Future
+import scala.util.{Failure, Success, Try}
+
+object RunControllerV3 {
+  final val LatestKey = "latest"
+}
+
+@RestController
+@RequestMapping(path = Array("/api-v3/runs"), produces = Array("application/json"))
+class RunControllerV3 @Autowired()(runService: RunServiceV3) extends BaseController {
+
+  import za.co.absa.enceladus.rest_api.utils.implicits._
+
+  import scala.concurrent.ExecutionContext.Implicits.global
+
+  @GetMapping()
```

Comment:
I have no idea where this happens, but the on `/api-v3/runs` using `startDate` filter I get an extra result that is not supposed to be there.

![image](https://user-images.githubusercontent.com/17467768/173776430-14ab16ab-3d44-45b1-a9db-2f664ec0398f.png)

Reply:
This issue has been addressed, the param now uses different date format `startDate=YYYY-MM-DD`